### PR TITLE
Add more victory types.

### PR DESCRIPTION
**Pull request by tmc**
https://formidable.com/open-source/victory/docs

### CODE REVIEW

**Comment by tmc** [comment]
It may be premature to call this '1.0.0' but I did make a technically breaking change on the styles attribute.

File: types/victory/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for Victory 0.9.3
+// Type definitions for Victory 1.0.0
```
Comment (DanielRosenwasser) [inline types/victory/index.d.ts]: Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]
So close! Let's just get linting working and we can pull this in.

File: types/victory/index.d.ts (hunk 1 above)
Comment (tmc) [reply types/victory/index.d.ts]: @DanielRosenwasser yep, 1m.

**Comment by bjarkehs** [comment]
Any chance you could also add VictorySelectionContainer to this? It's not currently exposed to Typescript. It's unfortunate that such a good charting library has such limited Typescript support.

**Pull request closed without merging**